skimage/util/_coord_conv.py

```diff
@@ -0,0 +1,234 @@
+import numpy as np
+
+
+def xy_to_rc(points=None, image=None):
+    """Convert points' coordinates and/or image from `x, y` to `r, c` convention.
+
+        o--->....       o--->....
+        |   x   .       |   c   .
+        |       .  ==>  |       .
+        v y     .       v r     .
+        .........       .........
+
+    Parameters
+    ----------
+    points : (K, 2) ndarray, optional
```

Comment:
I would call this `N, K` or `N, M`. (`K` is often used for number of dimensions.)